src/scripts.js

```diff
@@ -65,7 +67,7 @@ removeRecipeButton.addEventListener('click', removeRecipeFromFavorites)
 favoriteButton.addEventListener('click', displayFavoritesPage)
 pantryButton.addEventListener('click', displayPantryPage)
 addButton.addEventListener('click', addItemToPantry)
-cookRecipeButton.addEventListener('click', removeIngredientsFromPantry)
+cookRecipeButton.addEventListener('click', cookRecipe)
 searchBar.addEventListener('keypress', (event) => {
     if (event.key === "Enter" && homeView) {
```

Comment:
FYI when user is on pantry page, enters quantity and presses "Enter" it shoots them back to the home screen. We can add conditional here to check if we're on home view